pages/index.js

```diff
@@ -1,12 +1,42 @@
-import { useEffect } from 'react';
-import { useRouter } from 'next/router';
-import Layout from '@/components/Layout';
+import { useEffect } from "react";
+import { useRouter } from "next/router";
+import Layout from "@/components/Layout";
+import BannerLogo from "../public/Hack-R-Play/BannerLogo.png";
+import Banner from "@/components/Banner";
+import PastInitialives from "@/components/PastInitialives";
+
 
 export default function Home() {
 	const router = useRouter();
-	useEffect(() => {
-		router.push('/hackrplay/2022/home');
-	}, []);
+	// useEffect(() => {
+	// 	router.push('/hackrplay/2022/home');
+	// }, []);
+	const recentEvent={
```

Comment:
Instead of two objects, have one array with all the events. Each of the event object should have name, description, image, link, isInProgress properties. If isInProgress true, then it is current, else false.